src/rcms/fm/app/level1/HCALMasker.java

```diff
@@ -266,7 +268,10 @@ else if (functionManager.RunType.equals("global")){
               if (resourceName.contains(EvmTrigName)) {
                 //Mask all EvmTrig apps except for the ones we picked
                 if (!level2resource.getName().equals(eventBuilder) && !level2resource.getName().equals(trivialFU) && !level2resource.getName().equals(triggerAdapter)) { 
-                  allMaskedResources.add(new StringT(resourceName));
```

Comment:
In here, we add again the all un-picked EvmTrig Apps, which of course, are part of the maskedFM (if any) childResources. Hence the double-writing